src/Components/Dashboard/MiniDashboard.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { WithToken } from "../../HOCs/withToken";
+import { useEffect } from "react";
+import styled from "styled-components";
+import { useGetUser } from "../apis/useGetUser";
+import { set } from "lodash";
+import { useState } from "react";
+
+const MiniDashboardComponent = ({ user, setUser }) => {
+  const [res, getUser] = useGetUser()
+
+  useEffect(() => {
+    getUser()
+  }, [])
+
+  useEffect(() => {
+    if (res) {
```

Comment:
there will always be `res` as you defined initial value in the custom hook. what are we achieving here with this if?

Reply:
fixed

Reply:
how is this fixed ?